Yoast/wordpress-seo
# Improve performance for 7.7 upgrade routine

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the upgrade routing for 7.7 gave performance issues on larger databases.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Have posts with postmetas for the key `_yoast_wpseo_post_image_cache` - adding these manually. Perhaps the CLI can create large datasets for it (I don't know, it's just an idea)
* Install the Yoast test helper and sets the version to 7.6
* By setting the value, the version will updated directly
* Go to the database and verify that the postmetas for the key `_yoast_wpseo_post_image_cache` are deleted.

## Quality assurance


Fixes #10209
